packages/cli/src/zostso/start/as-app/StartASApp.handler.ts

```diff
@@ -32,6 +33,17 @@ export default class Handler extends ZosTsoBaseHandler {
             },
             this.mTsoStart
         );
-        commandParameters.response.console.log(JSON.stringify(response,null,2));
+        commandParameters.response.console.log(chalk.yellow.bold("\n" + "Servlet Key: ") + response.servletKey);
+        commandParameters.response.console.log(chalk.yellow.bold("Queue ID: ") + response.queueID + "\n");
+
+        response.tsoData.forEach((data) => {
+            if(typeof data === 'string') {
+                commandParameters.response.console.log(data);
+            } else if (data && data.DATA) {
+                commandParameters.response.console.log(data.DATA);
+            }
```

Comment:
Not a required change, but I like one-liners a lot 😋 
```suggestion
            commandParameters.response.console.log(typeof data === 'string' ? data : (data?.DATA ?? ""));
```